### ISSUE TITLE
WHAT IS APPROX

### ISSUE THREAD

**Issue by Galaxygingaf** [Humming-Owl/SMG2-Mod-Builder-To-USB-Loader#2]
The files I downloaded are all .rar

**Comment by Humming-Owl** [comment]
Is related to the size of the file containing the mod that you download, an approximate size.

**Comment by Galaxygingaf** [comment]
Sorry every time I put SMG64 Holiday Special v1.1.zip into the mods folder and then want to write to my SMG2.WBFS. He prompted me to make a mistake and asked me to download it again. Did I understand it wrong? That zip folder is indeed almost 43MB. Forgive me for being a noob indeed

**Comment by Humming-Owl** [comment]
Don't worry you are not supposed to know exactly what to do :)
I will see if I get any errors when doing it and then tell you later. Cheers.

**Comment by Humming-Owl** [comment]
I tried the tool and got no errors so something else is happening.
Just to verify, the WBFS file of the game and the mod file are in the correct locations? (as follows)

![1](https://user-images.githubusercontent.com/64061464/180622089-fba6ae30-a987-4385-a0b2-c877ff00ee66.png)
![2](https://user-images.githubusercontent.com/64061464/180622091-88ada556-415e-4041-b184-515f280e2fd2.png)

Also, the name of the mod file cannot be changed from original, the tool locates the mod file for its original name.

PD: I updated the tool (something I saw just now), so download it again and try.

Cheers.

**Comment by Galaxygingaf** [comment]
Yes, I found that it is because the upper case SMG2.WBFS will make errors, while when I have lower case letters, smg2.wbfs will not. But now I have a new problem, that is, there is an error in my 7z.dll. I don't know how to solve it. I wonder if it's because I'm windows10?
![微信图片_20220726105234](https://user-images.githubusercontent.com/109657721/180913184-6d54906d-cfee-43d2-a82f-b8ef2c5cc393.jpg)

**Comment by Galaxygingaf** [comment]
Thank you, it worked after adding the dll. I looked at the list and it seems that there is no Super Mario Starshine Demo v1.4, I think Starshine is also a very interesting SMG2 mod. This is the YouTube link: [https://youtu.be/LMkZmwOnQ2s](url) there is also the download address of the mod below the video introduction, can you please add it to the project.[https://www.mediafire.com/file/8ynmhr6v5y0owzb/Super_Mario_Starshine_Demo_v1.4.rar/file](url)

**Comment by Galaxygingaf** [comment]
Thank you, it worked after adding the dll. I looked at the list and it seems that there is no Super Mario Starshine Demo v1.4, I think Starshine is also a very interesting SMG2 mod. This is the YouTube link: https://youtu.be/LMkZmwOnQ2s, there is also the download address of the mod below the video introduction, can you please add it to the project.
download link：https://www.mediafire.com/file/8ynmhr6v5y0owzb/Super_Mario_Starshine_Demo_v1.4.rar/file

**Comment by Humming-Owl** [comment]
Glad it worked (I didn't knew that file was actually needed). I recently discovered the hack you mention and casually today I was able to incorporate it into the tool (it is a really awesome hack), hope it works correctly! Thanks for your feedback.

**Comment by Galaxygingaf** [comment]
I found a new SMG2 mod, but it looks like the original author doesn't plan on doing it anymore. I still saw it on the SPG64 channel, and he reposted it. Video link: https://youtu.be/DjL2Iihdgro
Download link: http://www.mediafire.com/file/z33ovufy0dgf0yn/Super_Mario_Galaxy_Deluxe_%255BRe-release%255D.rar/file

**Comment by Humming-Owl** [comment]
Your welcome. Nice to hear Starshine is working. The others that worked with SPG64 on Starshine are also really good at what they do. Have fun playing the hacks!
